03.asynchronous/callback.js

```diff
@@ -0,0 +1,27 @@
+import sqlite3 from "sqlite3";
+const db = new sqlite3.Database(":memory:");
+
+const createTableQuery = `CREATE TABLE books(
+    id INTEGER AUTO_INCREMENT PRIMARY KEY,
+    title VARCHAR NOT NULL UNIQUE
+)`;
+
+db.run(createTableQuery, function () {
+  db.run(
+    "INSERT INTO books (id, title) VALUES (?, ?)",
```

Comment:
`CREATE TABLE` とスタイルを合わせるなら以下のようになると思います。もしくは `CREATE TABLE` のほうのスタイルを調整してください。

```suggestion
    "INSERT INTO books(id, title) VALUES(?, ?)",
```

Reply:
`"INSERT INTO books(id, title) VALUES(?, ?)",`スペースなしの表記に変更しました。

[1ea8c58](https://github.com/smallmonkeykey/js-practices/pull/3/commits/1ea8c580dc319d9d9b78094681fa4c3bbdb8f469)